src/pages/registration/ContributorsPanel.tsx

```diff
@@ -19,21 +19,23 @@ const ContributorsPanel = () => {
   const contributorsError = errors.entityDescription?.contributors;
 
   return (
-    <BackgroundDiv backgroundColor={theme.palette.section.light}>
-      <Typography variant="h2">{t('contributors.authors')}</Typography>
-      <FieldArray name={ContributorFieldNames.CONTRIBUTORS}>
-        {({ push, replace, name }: FieldArrayRenderProps) => (
-          <>
-            <Authors push={push} replace={replace} />
-            {contributors.length === 0 && typeof contributorsError === 'string' && (
-              <FormHelperText error>
-                <ErrorMessage name={name} />
-              </FormHelperText>
-            )}
-          </>
-        )}
-      </FieldArray>
-    </BackgroundDiv>
+    <>
+      <BackgroundDiv backgroundColor={theme.palette.section.main}>
+        <Typography variant="h2" color="primary">
+          {t('contributors.authors')}
+        </Typography>
+        <FieldArray name={ContributorFieldNames.CONTRIBUTORS}>
+          {({ push, replace }: FieldArrayRenderProps) => <Authors push={push} replace={replace} />}
+        </FieldArray>
+      </BackgroundDiv>
+      {contributors.length === 0 && typeof contributorsError === 'string' && (
+        <BackgroundDiv backgroundColor={theme.palette.error.light}>
```

Comment:
Slipper du ikke å bruke `BackgroundDiv` sånn som `MuiFormHelper` er definert i theme nå? tolket det som at det skal holde med FormHelperText og error prop som er under

Reply:
jo, egentlig. La til bakgrunn for å få padding ihht skissene, men det er vel ikke helt ferdig enda uansett, så kan godt fjerne bakgrunnen her

Reply:
kan vel evt bruke SyledFormHelperText også, hvis det bare var for padding